diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/identifiers/LocalVariablesRule.kt

```diff
@@ -94,12 +96,28 @@ class LocalVariablesRule(private val configRules: List<RulesConfig>) : Rule("loc
     }
 
     private fun handleConsecutiveDeclarations(statement: PsiElement, properties: List<KtProperty>) {
+        val numLinesAfterLastProp = if (properties.last().node.treeNext.elementType == WHITE_SPACE) {
+            // minus one is needed to except \n after property
+            properties
+                    .last()
+                    .node
+                    .treeNext
+                    .numNewLines() - 1
+        } else {
+            0
+        }
+
         // need to check that properties are declared consecutively with only maybe empty lines
         properties
             .sortedBy { it.node.getLineNumber() }
-            .zip(properties.size - 1 downTo 0)
-            .forEach { (property, offset) ->
-                checkLineNumbers(property, statement.node.getLineNumber(), offset)
+            .let { it as MutableList }
```

Comment:
```suggestion
            .let { it.toMutableList() }
```
Also, why is it needed?